### DISCUSSION TITLE
twinBASIC BETA 64

### DISCUSSION THREAD

**Discussion by WaynePhillipsEA** [twinbasic/twinbasic#950] [Releases]
- improved: faster and more accurate form controls layout engine [ https://github.com/WaynePhillipsEA/twinbasic/issues/949 ]

<hr /><em>This discussion was created from the release <a href='https://github.com/WaynePhillipsEA/twinbasic/releases/tag/beta-64'>twinBASIC BETA 64</a>.</em>

**Comment by camomille22003** [comment]
it changes everything

![Animation16](https://user-images.githubusercontent.com/33807881/176209980-1838c3ca-6100-49fe-a9bf-162b975d9563.gif)

**Reply by bclothier** [reply]
@camomille22003  I'm not clear from the comment "it changes everything" --- is that a good thing or a bad thing? If there's a problem, maybe a comparison with how VB6 behaves will help clarify what is wrong.

**Reply by camomille22003** [reply]
compare with BETA 61 #939

**Comment by EduardoVB** [comment]
I think he meant it is working OK now.

**Comment by camomille22003** [comment]
Especially since it's a request I never made, but Wayne saw the problem in BETA 61 and fixed it in this version.

**Comment by EduardoVB** [comment]
I think we are not clear what was fixed then (or at least I am not), but good to know that "something" was fixed 🙂.
